mosviz/conftest.py

```diff
@@ -34,13 +34,45 @@
 #
 ## This is to figure out the affiliated package version, rather than
 ## using Astropy's
-# try:
-#     from .version import version
-# except ImportError:
-#     version = 'dev'
-#
-# try:
-#     packagename = os.path.basename(os.path.dirname(__file__))
-#     TESTED_VERSIONS[packagename] = version
-# except NameError:   # Needed to support Astropy <= 1.0.0
-#     pass
+import pytest
+
+try:
+    from .version import version
+except ImportError:
+    version = 'dev'
+
+try:
+    packagename = os.path.basename(os.path.dirname(__file__))
+    TESTED_VERSIONS[packagename] = version
+except NameError:   # Needed to support Astropy <= 1.0.0
+    pass
+
+@pytest.fixture(scope='session')
+def glue_gui():
+    import os
+
+    from glue.core import DataCollection
+    from glue.app.qt.application import GlueApplication
+    from glue.core import data_factories
+
+    from .viewers.mos_viewer import MOSVizViewer
+
+    # This is a deimos_mosviz.tbl file that is locally hosted
+    testdata = os.path.join("/Users/javerbukh/Documents/", "data_for_mosviz", "workshop_examples", "deimos", "deimos_mosviz.tbl")
```

Comment:
See `cubeviz` for the way to locate the data file based on a path relative to this source file:

https://github.com/spacetelescope/cubeviz/blob/f0c14bdb114e388cdd24878ba22f9fd43781479c/cubeviz/tests/helpers.py#L13

In order to make sure the data file is properly installed, you'll need to add a `setup_package.py` (which is part of the `astropy` package infrastructure) like this one in `cubeviz`:

https://github.com/spacetelescope/cubeviz/blob/master/cubeviz/tests/setup_package.py

You'll need to pay attention to the placement of this file relative to the actual data file in the directory layout. My suggestion would be to mirror the way that `cubeviz` does it and to place the data file within `mosviz/tests/data`. In this case, the `setup_package.py` file will go in `mosviz/tests`.

Reply:
@drdavella If I have the `deimos_mosviz.tbl` file in `mosviz.tests.data`, how should setup_package.py look? For now I have it like:
```python
def get_package_data():
    return {
        _ASTROPY_PACKAGE_NAME_ + '.tests': ['coveragerc'], 'tests.data': ['deimos_mosviz.tbl'],
    }
```